js/background.js

```diff
@@ -893,26 +933,26 @@ if (!SAFARI) {
 
             addMenu(translate("block_this_ad"), function(tab, clickdata) {
                 emit_page_broadcast(
-                    {fn:'top_open_blacklist_ui', options:{info: clickdata}},
-                    {tab: tab}
+                    { fn:"top_open_blacklist_ui", options: { info: clickdata } },
+                    { tab: tab }
```

Comment:
Spaces used in object literals but not array literals or function argument lists. Use one or the other.